intelmq/bots/outputs/smtp_batch/output.py

```diff
@@ -238,8 +238,7 @@ def prepare_mails(self):
                         continue
                 else:
                     # visible both warning and print
```

Comment:
with the print removed, the statement is likely not true any more

Reply:
Good point, fixed